Bipedal/bipedal_planner/bipedal_planner.py

```diff
@@ -0,0 +1,158 @@
+"""
+Bipedal Walking with modifying designated footsteps
+author: Takayuki Murooka (takayuki5168)
+"""
+import numpy as np
+import math
+from matplotlib import pyplot as plt
+import matplotlib.patches as pat
+from mpl_toolkits.mplot3d import Axes3D
+import mpl_toolkits.mplot3d.art3d as art3d
+
+class BipedalPlanner(object):
+    def __init__(self):
+        self.ref_footsteps = None
+        self.g = 9.8
+
+    def set_ref_footsteps(self, ref_footsteps):
+        self.ref_footsteps = ref_footsteps
+
+    def inverted_pendulum(self, x, x_dot, px_star, y, y_dot, py_star, z_c, time_width):
+        time_split = 100
+
+        for i in range(time_split):
+            delta_time = time_width / time_split
+            
+            x_dot2 = self.g / z_c * (x - px_star)
+            x += x_dot * delta_time
+            x_dot += x_dot2 * delta_time
+            
+            y_dot2 = self.g / z_c * (y - py_star)
+            y += y_dot * delta_time
+            y_dot += y_dot2 * delta_time
+
+            if i % 10 == 0:
+                self.com_trajectory.append([x, y])
+
+        return x, x_dot, y, y_dot
+        
+    def walk(self, T_sup=0.8, z_c=0.8, a=10, b=1, plot=False):
+        if self.ref_footsteps == None:
```

Comment:
should use the 'is' operator.

like:
if self.ref_footsteps is None:

https://lgtm.com/projects/g/AtsushiSakai/PythonRobotics/snapshot/723ee2368a5689bb10934d75b2befc827a807608/files/Bipedal/bipedal_planner/bipedal_planner.py?sort=name&dir=ASC&mode=heatmap#x7503d13671e38a27:1

Reply:
OK, I did it.